app/src/components/market/common/history_chart/index.tsx

```diff
@@ -83,6 +82,12 @@ const AnEvenSmallerLittleBall = styled(OutcomeItemLittleBallOfJoyAndDifferentCol
   width: 8px;
 `
 
+const AxisWrapper = styled.div`
+  display: inline;
+  stroke: ${props => props.theme.colors.verticalDivider};
+  fill: ${props => props.theme.colors.textColor};
```

Comment:
Looks like this still has old color values

Reply:
Yes, although the fact that this file being here is wrong (cause of my commits issues), shall we agree that the color values update will happen on a later single issue @Mi-Lan @pimato ?